platform-includes/configuration/data-to-all-spans/unity.mdx

```diff
@@ -0,0 +1,22 @@
+```csharp
+SentrySdk.Init(options =>
+{
+    options.SetBeforeSendTransaction(transaction =>
+    {        
+        // Set the attribute on the root span
+        if (transaction.Contexts.Trace == null)
+        {
+            transaction.Contexts.Trace = new SpanContext();
+        }
```

Comment:
Do we need the `null` check?

Not nullable according to: https://github.com/getsentry/sentry-dotnet/blob/b1d397211d0070665725ea003f1c4356b1748ff9/src/Sentry/SentryContexts.cs#L76C27-L76C43

```suggestion
        // Set the attribute on the root span
```